tron/kubernetes.py

```diff
@@ -433,7 +437,23 @@ def recover(self, task: KubernetesTask) -> None:
         """
         Given an instance of a KubernetesTask, attempt to reconcile the current state of the task from Kubernetes.
         """
-        pass
+        if not task:
+            return
+
+        if not self.enabled:
+            task.log.info("Could not recover task, Kubernetes usage is disabled.")
+            task.exited(None)
+            return
+
+        self._check_connection()
+
+        # pod name
+        task_id = task.get_kubernetes_id()
```

Comment:
we probably want a comment that's a little more descriptive, maybe: 
```suggestion
        # the task/kubernetes id is really just the pod name
        task_id = task.get_kubernetes_id()
```